inc/saiobject.h

```diff
@@ -37,6 +37,8 @@
 #include <saisrv6.h>
 
 /* new experimental object type includes */
+#include <saiexperimentaldashtunnel.h>
+#include <saiexperimentaldashha.h>
```

Comment:
no need for this

Reply:
Why not? Here the list has included `saiexperimentaldashappliance.h, saiexperimentaldashflow.h, saiexperimentaldashmeter.h, saiexperimentaldashvip.h`, etc. The list is automatically updated while `make sai-headers` in DASH to have header files of all dash objects.

Reply:
since no item is used from included files, so including them is redundant